exploration-module/src/main/java/io/edpn/backend/exploration/adapter/persistence/entity/mapper/MybatisStationEntityMapper.java

```diff
@@ -0,0 +1,84 @@
+package io.edpn.backend.exploration.adapter.persistence.entity.mapper;
+
+import io.edpn.backend.exploration.adapter.persistence.entity.MybatisStationEntity;
+import io.edpn.backend.exploration.adapter.persistence.entity.MybatisSystemEntity;
+import io.edpn.backend.exploration.application.domain.LandingPadSize;
+import io.edpn.backend.exploration.application.domain.Station;
+import io.edpn.backend.exploration.application.dto.persistence.entity.StationEntity;
+import io.edpn.backend.exploration.application.dto.persistence.entity.mapper.StationEntityMapper;
+import io.edpn.backend.exploration.application.dto.persistence.entity.mapper.SystemEntityMapper;
+import lombok.RequiredArgsConstructor;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+@RequiredArgsConstructor
+public class MybatisStationEntityMapper implements StationEntityMapper<MybatisStationEntity> {
+
+    private final SystemEntityMapper<MybatisSystemEntity> systemEntityMapper;
+
+    private static Map<String, Integer> getLandingPads(Station station) {
```

Comment:
I know this is a matter of preference/taste, but personally I'd totally opt for the Stream API for stuff like this.

Something like:
```
return LandingPadSize.values().stream()
    .filter(landingPadSize -> !landingPadSize.equals(LandingPadSize.UNKNOWN))
    .collect(Collectors.toMap(
        Function.identity(),
        landingPadSize -> station.landingPads().getOrDefault(landingPadSize, 0)
    ));
```

Reply:
Oops, that would have to be `Stream.of(landingPadSize.values())`.

Reply:
rewrote it to this
```
    private static Map<String, Integer> getLandingPads(Station station) {
        return LandingPadSize.getKnownLandingPadSizes().stream()
                .collect(Collectors.toMap(
                        LandingPadSize::name,
                        landingPadSize -> station.landingPads().getOrDefault(landingPadSize, 0)));
    }
```